samcli/local/init/templates/cookiecutter-aws-sam-hello-powershell/Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
```

Comment:
You can delete this file as we will not run integ tests for these templates as of now :)